persistence/state.go

```diff
@@ -354,11 +354,39 @@ func (p *PostgresContext) updateTransactionsTree() error {
 }
 
 func (p *PostgresContext) updateParamsTree() error {
-	// TODO(#361): Create a core starter task to implement this
+	paramsOrFlags, err := p.getParamsOrFlags(types.ParamsTableName, p.Height, false)
```

Comment:
Let's avoid having `state.go` have knowledge of the table names and.

1. We want to avoid having `state.go` have knowledge of implementation details such as the table name. 

2. Change these calls to `getParamsUpdate(height)` and `getFlagsUpadted(height)`.

3. Since the tree maintains a state, the order in which the results are returned should not affect the root hash, so we shouldn't need to pass that variable here.